es/03_5_Entendiendo_El_Descriptor.md

```diff
@@ -0,0 +1,157 @@
+  "iswatchonly": false,
+  "isscript": false,
+  "iswitness": false,
+  "pubkey": "03efdee34c0009fd175f3b20b5e5a5517fd5d16746f2e635b44617adafeaebc388",
+  "iscompressed": true,
+  "ischange": false,
+  "timestamp": 1592335136,
+  "hdkeypath": "m/0'/0'/18'",
+  "hdseedid": "fdea8e2630f00d29a9d6ff2af7bf5b358d061078",
+  "hdmasterfingerprint": "d6043800",
+  "labels": [
+    ""
+  ]
+}
+```
+
+Aquí el descriptor es `pkh ([d6043800 / 0 '/ 0' / 18 '] 03efdee34c0009fd175f3b20b5e5a5517fd5d16746f2e635b44617adafeaebc388) # 4ahsl9pk`.
+
+## Entender un descriptor
+
+Un descriptor se divide en varias partes:
+
+```
+function([derivation-path]key)#checksum
+```
+
+Esto es lo que todo eso significa:
+* **Función.** La función que se utiliza para crear una dirección a partir de esa tecla. En estos casos, es `pkh`, que es la dirección heredada P2PKH estándar que conoció en [§3.3: Configuración de su billetera](03_3_Setting_Up_Your_Wallet.md). De manera similar, una dirección P2WSH SegWit usaría `wsh` y una dirección P2WPKH usaría` wpkh`.
+* **Ruta de derivación.** Esto describe qué parte de una billetera HD se está exportando. En este caso, es una semilla con la huella digital `d6043800` y luego el hijo 18 del hijo 0 del hijo 0 (` 0'/ 0'/18'`) de esa semilla. También puede haber una derivación adicional después de la clave: `función ([ruta de derivación] clave / más-derivación) # suma de comprobación`
+   * Vale la pena señalar aquí que si alguna vez obtiene una ruta de derivación sin una huella digital, puede inventarla. Es solo que si hay uno existente, debe coincidir, porque si alguna vez regresa al dispositivo que creó la huella digital, deberá tener el mismo.
+* **Clave**. La clave o claves que se están transfiriendo. Esto podría ser algo tradicional como un `xpub` o` xprv`, podría ser simplemente una clave pública para una dirección, como en este caso, podría ser un conjunto de direcciones para una firma múltiple, o podría ser otra cosa. Estos son los datos centrales: la función explica qué hacer con ellos.
+* **Suma de comprobación**. Los descriptores están destinados a ser transferibles por humanos. Esta suma de comprobación asegura que lo haga bien.
+
+Consulte [Información de Bitcoin Core sobre la compatibilidad con descriptores](https://github.com/bitcoin/bitcoin/blob/master/doc/descriptors.md) para obtener más información.
+
+## Examine un Descriptor
+
+Usted puede examinar el descriptor con el comando RPC `getdescriptorinfo`:
+```
+$ bitcoin-cli getdescriptorinfo "pkh([d6043800/0'/0'/18']03efdee34c0009fd175f3b20b5e5a5517fd5d16746f2e635b44617adafeaebc388)#4ahsl9pk"
+{
+  "descriptor": "pkh([d6043800/0'/0'/18']03efdee34c0009fd175f3b20b5e5a5517fd5d16746f2e635b44617adafeaebc388)#4ahsl9pk",
+  "checksum": "4ahsl9pk",
+  "isrange": false,
+  "issolvable": true,
+  "hasprivatekeys": false
+}
+```
+Tenga en cuenta que devuelve una suma de comprobación. Si alguna vez le dan un descriptor sin una suma de verificación, puede aprenderlo con este comando:
```

Comment:
- `...puede aprenderla...`. (Aprender la suma de verificacion).

Reply:
I don't think this is correct,   We're talking about the fact to learning, not the checksum itself.